src/EventSubscriber/CachePurge/ContentEventsSubscriber.php

```diff
@@ -132,4 +145,16 @@ public function onUpdateContentMetadataEvent(UpdateContentMetadataEvent $event):
             $this->getContentTags($contentId)
         );
     }
+
+    private function isContentTypeFullyTranslatable(ContentType $contentType): bool
+    {
+        /** @var \eZ\Publish\API\Repository\Values\ContentType\FieldDefinition $fieldDefinition */
+        foreach ($contentType->getFieldDefinitions() as $fieldDefinition) {
+            if (!$fieldDefinition->isTranslatable) {
+                return false;
+            }
+        }
+
+        return true;
```

Comment:
Coulkd be replaced with `\eZ\Publish\API\Repository\Values\ContentType\FieldDefinitionCollection::any` call